tools/codegen/api/lazy.py

```diff
@@ -118,7 +122,8 @@ def __init__(self, func: FunctionSchema):
 
     @property
     def node_name(self) -> str:
-        return str(self.name).lower().capitalize()
+        op_name = str(self.name).lower()
+        return ''.join(word.capitalize() or '' for word in op_name.split('_'))
```

Comment:
what's the `or ''` for?  These seem the same to me, but i'm probably missing some case:

```
>>> [word.capitalize() or '' for word in "_log_softmax".split("_")]
['', 'Log', 'Softmax']
>>> [word.capitalize() for word in "_log_softmax".split("_")]
['', 'Log', 'Softmax']
```

Reply:
The `''` replaces any occurrences so we get `LogSoftmax` instead of `_LogSoftmax`.